packages/babel-cli/src/babel/dir.js

```diff
@@ -71,7 +71,11 @@ module.exports = function (commander, filenames) {
     _.each(filenames, function (dirname) {
       let watcher = chokidar.watch(dirname, {
         persistent: true,
-        ignoreInitial: true
+        ignoreInitial: true,
+        awaitWriteFinish: {
+          stabilityThreshold: 50,
+          pollInterval: 10,
```

Comment:
This seems like a pretty fast interval, perhaps we should do more like `200` and `50`?